src/en/rokaricomics/src/eu/kanade/tachiyomi/extension/en/rokaricomics/RokariComics.kt

```diff
@@ -0,0 +1,22 @@
+package eu.kanade.tachiyomi.extension.en.rokaricomics
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+
+class RokariComics : MangaThemesia(
+    "RokariComics",
+    "https://rokaricomics.com",
+    "en",
+) {
+    override fun getFilterList(): FilterList {
+        return FilterList(
+            Filter.Separator(),
+            StatusFilter(intl["status_filter_title"], statusOptions),
+            TypeFilter(intl["type_filter_title"], typeFilterOptions),
+            OrderByFilter(intl["order_by_filter_title"], orderByFilterOptions),
+            Filter.Header(intl["genre_exclusion_warning"]),
+            GenreListFilter(intl["genre_filter_title"], getGenreList()),
```

Comment:
```suggestion
        if (!genrelist.isNullOrEmpty()) {
            filters.addAll(
                listOf(
                    Filter.Header(intl["genre_exclusion_warning"]),
                    GenreListFilter(intl["genre_filter_title"], getGenreList()),
                ),
            )
        } else {
            filters.add(
                Filter.Header(intl["genre_missing_warning"]),
            )
        }
```
may need to adjust lint